apis/openstack-heat/src/main/java/org/jclouds/openstack/heat/v1/domain/Attributes.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * 
+ */
+package org.jclouds.openstack.heat.v1.domain;
+
+import java.beans.ConstructorProperties;
+import java.util.Set;
+
+import javax.inject.Named;
+
+import org.jclouds.openstack.v2_0.domain.Link;
+import org.jclouds.openstack.v2_0.domain.Resource;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Objects.ToStringHelper;
+
+/**
+ * @author deviyada
+ * 
+ */
+public class Attributes extends Resource {
+
+	private String progress;
+	private String status;
+	@Named("config_drive")
+	private String configDrive;
+	@Named("tenant_id")
+	private String tenantId;
+	@Named("host_id")
+	private String hostId;
+	private String created;
+	@Named("user_id")
+	private String userId;
+
+	@ConstructorProperties({ "id", "name","links","progress", "status", "config_drive",
+			"tenant_id", "host_id", "created", "user_id"})
+	public Attributes(String id,String name, Set<Link> links, String progress, String status,
+			String configDrive, String tenantId, String hostId,String created, String userId) {
+		super(id, name, links);
+		this.progress = progress;
+		this.status = status;
+		this.configDrive = configDrive;
+		this.tenantId = tenantId;
+		this.hostId = hostId;
+		this.created = created;
+		this.userId = userId;
+	}
+
+	public String getProgress() {
```

Comment:
Please add comment with the return value for for example :
  /**
    * @return the status of this Stack.
    */